### DISCUSSION TITLE
Rename project [Jun 09]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2402] [Discord Threads]
> 🙋‍♂️ **Gi**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Not at the moment  
#  
> 🙋‍♂️ **Gi**  
#### Alright thank you  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/984007353106067456)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
